react-playground/src/components/Header.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import styled from "styled-components";
+import MainLogo from "../images/main-logo.svg";
+import MenuIcon from "../images/menu-icon.svg";
+
+const HeaderWrapper = styled.div`
```

Comment:
I imagine the styled components are meant to go in a separate file. Do you create files for each individual component or do they live together?

Reply:
Yeah that makes sense. Usually we do a `src/components/Header/Header.tsx` type of ordering. We also have an `src/app/components` for more platform specific components as well, but the former option should work for your case.

Reply:
Inside of that Header/ folder, beside the component, we also include an index.ts with `export { default as Card } from './Card';` to enable simpler import statements when you need the component, like: `import { Header } from 'components/Header`

Reply:
You can reference the starter kit for some examples https://github.com/8thlight/react-ui-starter-kit/tree/main/src/components